conf/maestro-conf/edge-config-dell5000-demo.yaml

```diff
@@ -29,34 +31,38 @@ platform_readers:
 #     This is only a valid option if you are not using a well-known CA
 #    server_name: "devcloud.wigwag.io"
 var_defs:
+   - key: "SNAP_DATA"
+     value: "/var/snap/pelion-edge/current"
+   - key: "SNAP"
+     value: "/snap/pelion-edge/current"
```

Comment:
Making sure, does this value also need a `/var` prefix?

Reply:
I think that looks correct, SNAP should point to /snap which is the read-only partition.